tools/cli/go-whisk-cli/commands/commands.go

```diff
@@ -34,22 +33,21 @@ var client *whisk.Client
 func setupClientConfig(cmd *cobra.Command, args []string) (error){
     baseURL, err := getURLBase(Properties.APIHost)
 
-    // Don't display the 'invalid apihost' error if this CLI invocation is setting that value
-    isApiHostPropSetCmd := cmd.Parent().Name() == "property" && cmd.Name() == "set" && len(flags.property.apihostSet) > 0
-    if err != nil && !isApiHostPropSetCmd {
+    isPropertyGet := cmd.Parent().Name() == "property" && cmd.Name() == "get" && (flags.property.auth ||
+      flags.property.apihost || flags.property.namespace || flags.property.apiversion || flags.property.cliversion)
+
+    isPropertySet := cmd.Parent().Name() == "property" && cmd.Name() == "set" && (len(flags.property.apihostSet) > 0 ||
+    len(flags.property.namespaceSet) > 0 || len(flags.property.apiversionSet) > 0 || len(flags.global.auth) > 0)
```

Comment:
@dubeejw - Thinking out loud... Does it make sense to ignore the invalid API host value when setting property values other than the API host?  When the namespace is set, the CLI communicates with the OW host to validate the namespace.  I'm also thinking that both the API version and auth key should really only be set if there's a valid OW API host configured.  The only exception would be the setting of the API host property itself.

When the API host property is invalid, that seems to be a pretty important bit of configuration to correct ASAP.